doc/api/child_process.md

```diff
@@ -1075,6 +1075,13 @@ When the child process is a Node.js instance (e.g. spawned using
 [`child_process.fork()`][]), the `process.disconnect()` method can be invoked
 within the child process to close the IPC channel as well.
 
+###subprocess.exitCode:
```

Comment:
```suggestion
### `subprocess.exitCode`
```

Reply:
@addaleax , thanks. Fixed it.

Reply:
@HarshithaKP There is still a missing space after the `###` everywhere.

Reply:
@addaleax, thanks. I missed it in last commit. Now fixed it.